src/mpyl/utilities/s3/__init__.py

```diff
@@ -15,17 +15,18 @@
 
 @dataclass
 class S3ClientConfig:
-    service_name = "s3"
-    region_name = "eu-central-1"
-    access_key_id: Optional[str]
-    secret_access_key: Optional[str]
     bucket_name: str
     bucket_root_path: str
+    service_name: str = field(default="s3")
```

Comment:
why do we need `field` compared to just `service_name: str = "s3"` ?

Reply:
we don't, I had just been looking at the dataclass docs and saw it, but it's not necessary - I removed it